config/environments/production.rb

```diff
@@ -31,10 +31,12 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = Uglifier.new(harmony: true)
```

Comment:
Was this removed to test the docker image?

Reply:
@bodhish I'm not completely sure now, but I think this was removed because this isn't a part of Rails 6's default `production.rb` file, and could have made a JS runtime necessary for the app to boot up. I remember that I'd struggled for a while to get the app to boot up in a pure Ruby environment.